db/views/reporting_facility_states_v11.sql

```diff
@@ -0,0 +1,355 @@
+               COUNT(distinct(patient_id)) FILTER (WHERE htn_care_state = 'under_care' AND diabetes_treatment_outcome_in_last_3_months = 'bs_over_300') AS bs_over_300_under_care,
+               COUNT(distinct(patient_id)) FILTER (WHERE htn_care_state = 'under_care' AND diabetes_treatment_outcome_in_last_3_months = 'missed_visit') AS bs_missed_visit_under_care,
+               COUNT(distinct(patient_id)) FILTER (WHERE htn_care_state = 'under_care' AND diabetes_treatment_outcome_in_last_3_months = 'visited_no_bs') AS visited_no_bs_under_care,
+
+               COUNT(distinct(patient_id)) FILTER (WHERE htn_care_state = 'lost_to_follow_up' AND diabetes_treatment_outcome_in_last_3_months = 'missed_visit') AS bs_missed_visit_lost_to_follow_up,
+
+               COUNT(distinct(patient_id)) FILTER (WHERE htn_care_state = 'under_care') AS diabetes_patients_under_care,
+               COUNT(distinct(patient_id)) FILTER (WHERE htn_care_state = 'lost_to_follow_up') AS diabetes_patients_lost_to_follow_up
+
+        FROM reporting_patient_states
+        WHERE diabetes = 'yes'
+          AND months_since_registration >= 3
+        GROUP BY 1, 2
+    ),
+
+    monthly_cohort_outcomes AS (
+        SELECT assigned_facility_region_id AS region_id, month_date,
+               COUNT(distinct(patient_id)) FILTER (WHERE htn_treatment_outcome_in_last_2_months = 'controlled') AS controlled,
+               COUNT(distinct(patient_id)) FILTER (WHERE htn_treatment_outcome_in_last_2_months = 'uncontrolled') AS uncontrolled,
+               COUNT(distinct(patient_id)) FILTER (WHERE htn_treatment_outcome_in_last_2_months = 'missed_visit') AS missed_visit,
+               COUNT(distinct(patient_id)) FILTER (WHERE htn_treatment_outcome_in_last_2_months = 'visited_no_bp') AS visited_no_bp,
+               COUNT(distinct(patient_id)) AS patients
+
+        FROM reporting_patient_states
+        WHERE hypertension = 'yes'
+          AND months_since_registration = 2
+        GROUP BY 1, 2
+    ),
+
+    monthly_overdue_calls AS (
+        SELECT appointment_facility_region_id AS region_id, month_date,
+               COUNT(distinct(appointment_id)) AS call_results
+        FROM reporting_overdue_calls
+        GROUP BY 1, 2
+    ),
+
+    monthly_follow_ups AS (
+        SELECT facility_id, month_date, COUNT(distinct(patient_id)) as follow_ups
+        FROM reporting_patient_follow_ups
+        WHERE hypertension = 'yes'
+        GROUP BY facility_id, month_date
+    ),
+
+    monthly_diabetes_follow_ups AS (
+        SELECT facility_id, month_date, COUNT(distinct(patient_id)) as follow_ups
+        FROM reporting_patient_follow_ups
+        WHERE diabetes = 'yes'
+        GROUP BY facility_id, month_date
+    ),
+
+    monthly_overdue_patients as (
+        SELECT assigned_facility_region_id as region_id, month_date,
+           COUNT(*) filter(where ltfu = 'no' and is_overdue = 'yes')  as overdue_patients,
```

Comment:
We'll need the sum of assigned patients (+contactale) no?

Reply:
The `assigned_patient` count is already there in reporting_facility_states. We could use that.